GitUI/CommandsDialogs/SettingsDialog/Pages/DiffViewerSettingsPage.cs

```diff
@@ -1,14 +1,26 @@
 ﻿using GitCommands;
+using ResourceManager;
 
 namespace GitUI.CommandsDialogs.SettingsDialog.Pages
 {
     public partial class DiffViewerSettingsPage : SettingsPageWithHeader
     {
+        private readonly TranslationString _showDiffForAllParents = new TranslationString(@"If set, show all differences (not just the first differences).
+
+For a single selected commit, show the difference to the parent commit.
+For a single selected merge commit, show all parents.
+
+If up to four commits are multi-selected, show the difference from the first selected commits to the last selected commit.
+(If more than four commits are selected, show the difference from the first to the last selected commit only).
+
+If two commits are selected, also show the difference from the common ancestor (BASE) to the last selected.");
+
```

Comment:
Overall, I'd word it something like this (subject to other questions):

```
Show all differences between selected commits.

- For a single selected commit, show a difference between it and its parent commit.
- For a single selected merge commit, show all parents.
- For two selected commits, show a difference between a common ancestor (BASE) to the last selected commit.
- For multiple selected commits (up to 4), show a difference from the first selected to the last selected commit.
- For multiple selected commits (more than 4), show a difference between the first and the last selected commits only.
```

Reply:
I updated some more